deltachat-ios/Helper/GiveBackMyFirstResponder.swift

```diff
@@ -1,4 +1,5 @@
 import UIKit
+import Combine
```

Comment:
What do you need Combine for? I'm a little confused as I didn't see any code using it :)

Reply:
Whoops, left over from experiment. Will remove.